src/style.css

```diff
@@ -1 +1,60 @@
 /* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  padding: 10px 50px;
+}
+
+.navBar {
+  display: flex;
+}
+
+.navigation {
+  display: flex;
+  gap: 10px;
+  align-items: center;
+}
```

Comment:
I can't understand what the difference between these classes `navBar` and `navigation`. Use more meaningful names